src/compiler/compile/nodes/Attribute.js

```diff
@@ -64,10 +66,22 @@ export default class Attribute extends Node {
 						}
 				  );
 		}
+
 		if (this.dependencies.size > 0) {
 			parent.cannot_use_innerhtml();
 			parent.not_static_content();
 		}
+
+		// TODO Svelte 5: Think about moving this into the parser and make it an error
```

Comment:
or should we merge this PR in the Svelte 3 branch and make it an error in Svelte 4?